src/components/Article.jsx

```diff
@@ -1 +1,39 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </div>
+  );
+};
+
+Article.PropTypes = {
+  title: PropTypes.string.isRequired,
+  date: PropTypes.number.isRequired,
```

Comment:
The mistake of linter here, and I guess, that we have date type like string, not number

Reply:
In general, really good code!)